rasa/cli/utils.py

```diff
@@ -197,5 +199,49 @@ def payload_from_button_question(button_question: "Question") -> Text:
 
 
 def signal_handler(sig, frame) -> NoReturn:
+    """Signal Handler from Rasa X.
+
+    Args:
+        sig: The signal number.
+        frame: The interrupted stack frame.
+    """
     print("Goodbye 👋")
     sys.exit(0)
+
+
+def get_valid_config(
```

Comment:
a function prefixed with `get_` implies a getter function without side effects. The fact that this can do `sys.exit` seems bit dangerous to me.

Reply:
What would you recommend - changing its name to sth like `get_valid_config_or_exit` or a change of logic (perhaps with a separate issue)? FWIW, this previously was a private method and I've made it public because its now used in various places. I wouldn't feel entirely comfortable in changing the logic as this will affect behaviour and code elsewhere that has nothing to do with this PR.

Reply:
mhm, valid point. Renaming it would then be the best option! 👍🏻